magento/inventory
# MSI-2522 improve result output for reservation cli

### Description (*)
Introduce bunching logic for reservation inconsistency check CLI

Tests with 1255 orders (with default bunch size: 50):
- duration: 18.30s
- memory usage: 50.25MB

Memory usage can be adjusted by bunch size option.

### Fixed Issues (if relevant)

1. magento/inventory#2522: Inventory reservation list inconsistencies command does not work

### Manual testing scenarios (*)
1. Create orders with missing reservations
2. Run `bin/magento inventory:reservation:list-inconsistencies` - the command returns now results in bunches (default bunch size = 50)
3. Run `bin/magento inventory:reservation:list-inconsistencies --bunch-size 2` to set bunch size to 2
4. Test piping `bin/magento inventory:reservation:list-inconsistencies --bunch-size 2 --raw | bin/magento inventory:reservation:create-compensations` to verify that piping still works

### Contribution checklist (*)
